ERR_PNPM_NO_OFFLINE_TARBALL while bulding off main

```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/w7gm2zyz0k6rdz281809jkpzlf0h3d91-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Executing pnpmConfigHook
Configuring pnpm store
Installing dependencies
Scope: all 2 workspace projects
Lockfile is up to date, resolution step is skipped
Progress: resolved 1, reused 0, downloaded 0, added 0
Packages: +544

Progress: resolved 1, reused 0, downloaded 0, added 0
Progress: resolved 544, reused 10, downloaded 0, added 0
 ERR_PNPM_NO_OFFLINE_TARBALL  A package is missing from the store but cannot download it in offline mode. The missing package may be downloaded from https://registry.npmjs.org/js-yaml/-/js-yaml-4.1.0.tgz.
Progress: resolved 544, reused 10, downloaded 0, added 0
```

--- a/vencord.nix
+++ b/vencord.nix
@@ -1,3 +1,7 @@
+# Made from nixpkgs pkg and updated with nix-update
+# identical to nixpkgs source, but maintained here for
+# quicker updates that don't wait on hydra
+
 {
   curl,
   esbuild,
@@ -26,7 +30,7 @@ stdenv.mkDerivation (finalAttrs: {
   pnpmDeps = pnpm.fetchDeps {
     inherit (finalAttrs) pname src;
 
-    hash = "sha256-bosCE9gBFCcM3Ww6sJmhps/cl4lovXKMieYpkqAMst8=";
+    hash = "sha256-YBWe4MEmFu8cksOIxuTK0deO7q0QuqgOUc9WkUNBwp0=";
   };
 
   nativeBuildInputs = [

